src/commons/commands.py

```diff
@@ -22,8 +22,20 @@
 """All commands for corio."""
 
 # K8s commands for cortx.
-CMD_K8S_WORKERS_NAME = "kubectl get nodes -l node-role.kubernetes.io/worker=worker|awk '{print $1}'"
 # Parameters: pod_name, container_name, command.
+KUBECTL_CMD = "kubectl {} {} -n {} {}"
+CMD_K8S_WORKERS_NAME = "kubectl get nodes -l node-role.kubernetes.io/worker=worker|awk '{print $1}'"
```

Comment:
Support needs to added for the namespace changes.

Reply:
May you please elaborate

Reply:
@VikasKumarC  Deployment is done in specific namespace, So the kubectl commands needs to have namespace specified or the context should be set before starting the test.

Reply:
Hi @priyankaborawake 
command is formatted with cmd = commands.KUBECTL_CMD.format_map(operation, pod, **namespace**, command_suffix)
namespace is sent to method 
self.send_k8s_cmd(
                        operation="exec", pod=pod, 
                        **namespace=const.NAMESPACE,**
                        command_suffix=f"-c {cnt} -- sync", decode=True)

Reply:
So this const.NAMESPACE if different than default then it will need to set as current namespace context before running any kubectl commands on ur mastre node. Command is:
KUBECTL_SET_CONTEXT = "kubectl config set-context --current --namespace={}"
May u need to set it when u r doing any client config before running any tests